quantum_systems/time_evolution_operators/operator.py

```diff
@@ -55,7 +63,7 @@ def h_t(self, current_time):
             time-point.
         """
 
-        return self._system.h
+        return 0
```

Comment:
Why are this returning 0 instead of self._system.h?

Reply:
This is because the time-evolution operators themselves should no longer add the time-independent one- and two-body Hamiltonian. For example, with the old solution two `DipoleFieldInteraction`-operators would add  the time-independent `h` and `u` twice to the time-evolution ([old dipole operator](https://github.com/Schoyen/quantum-systems/blob/master/quantum_systems/time_evolution_operators/operator.py#L135) vs [new operator](https://github.com/Schoyen/quantum-systems/blob/multiple-time-evolution-operators/quantum_systems/time_evolution_operators/operator.py#L143)). The user should now instead specify if the time-independent one- and two-body operators should be added to the time-evolution in [`set_time_evolution_operator`](https://github.com/Schoyen/quantum-systems/blob/multiple-time-evolution-operators/quantum_systems/system.py#L139-L170). This is then done in [`h_t`](https://github.com/Schoyen/quantum-systems/blob/multiple-time-evolution-operators/quantum_systems/system.py#L185-L189) and [`u_t`](https://github.com/Schoyen/quantum-systems/blob/multiple-time-evolution-operators/quantum_systems/system.py#L199-L203) in `QuantumSystem`.